com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -744,10 +744,6 @@ static void InitializeShadowData(UniversalRenderPipelineAsset settings, NativeAr
             shadowData.bias = m_ShadowBiasData;
             shadowData.supportsMainLightShadows = SystemInfo.supportsShadows && settings.supportsMainLightShadows && mainLightCastShadows;
 
-            // We no longer use screen space shadows in URP.
-            // This change allows us to have particles & transparent objects receive shadows.
-            shadowData.requiresScreenSpaceShadowResolve = false;// shadowData.supportsMainLightShadows && supportsScreenSpaceShadows && settings.shadowCascadeOption != ShadowCascadesOption.NoCascades;
```

Comment:
Is this initialized to false anywhere else? otherwise we might have garbage here.

Reply:
Let me restore it and add 'obsolete' attribute, thanks